uportal-war/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ *  Copyright 2003-2010 Terracotta, Inc.
```

Comment:
Good question. This source does need to be cleaned up. It's a copy from ehcache-jgroups3-replication: https://github.com/Jasig/ehcache-jgroups3-replication/blob/master/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

The idea is to merge this in to reduce dependencies. It really should move into the uPortal 5 cache module, if there is such a beast.